src/CoreWCF.Primitives/src/CoreWCF/Channels/TextMessageEncoderFactory.cs

```diff
@@ -90,7 +90,7 @@ internal static string GetMediaType(MessageVersion version)
             return mediaType;
         }
 
-        internal static string GetContentType(string mediaType, Encoding encoding)
+        public static string GetContentType(string mediaType, Encoding encoding)
         {
```

Comment:
I'm not seeing any usage outside of Primitives for this method of GetEncodingFromContextType. Can you revert to internal.